style: apply /style-guide pass to weave (#2739)
wandb/docs: weave/guides/integrations/openai.mdx

---
title: "OpenAI"
description: "Integrate OpenAI with Weave for tracing, evaluation, and monitoring"
keywords: [GPT-4, Assistants API, Batch API, structured outputs, function calling]
---

<a target="_blank" href="https://colab.research.google.com/github/wandb/examples/blob/master/weave/docs/quickstart_openai.ipynb" aria-label="Open in Google Colab">
  <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>
</a>

This guide shows you how to integrate the OpenAI Python and TypeScript libraries with Weave so that you can trace, evaluate, and monitor your LLM application. It's intended for developers who already use OpenAI's SDKs and want visibility into their calls during development and in production.

<Note>

> Do you want to experiment with OpenAI models on Weave without any setup? Try the [LLM Playground](../tools/playground).

Reviewer: ```suggestion
Experiment with OpenAI models on Weave without any setup using the [LLM Playground](../tools/playground).
```

Author: Applied verbatim in eaf82665d.
